rules/windows/process_creation/win_powershell_disable_windef_av.yml

```diff
@@ -0,0 +1,26 @@
+title: Powershell Used To Disable Windows Defender AV Security Monitoring
+id: a7ee1722-c3c5-aeff-3212-c777e4733217
+status: experimental
+description: Detects attackers attempting to disable Windows Defender using Powershell
+author: ok @securonix invrep-de, oscd.community
```

Comment:
Hi @invrep-de !
Please add `'` around string values.
It is also better to split the second rule into a new Pull Request.